uber/templates/base.html

```diff
@@ -45,6 +45,27 @@
     </style>
     <script>
         var DISABLE_STRIPE_BUTTONS_ON_CLICK = true;
```

Comment:
This is great! I would like a later refactor to move these definitions into a config-related location just to make them easier to find/modify, but you've done the frontend work needed for that so it'll be easy. Theoretically.